src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/CyclesPayoutsPanel/hooks/useV4UpcomingCycleHasChanges.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from "react"
+import { useV4CurrentUpcomingConfigurationPanel } from "./useV4CurrentUpcomingConfigurationPanel"
+
+export const useV4UpcomingCycleHasChanges = () => {
+  const data = useV4CurrentUpcomingConfigurationPanel('upcoming')
+
+  const loading = useMemo(
```

Comment:
In general we can chill out a bit on useMemo. In fact we shouldn't use useMemo until we have a clear performance issue that we need to address with it.

Reply:
Ok got it haha, this was just copied code.